support-frontend/assets/pages/digital-subscription-landing/components/feedbackWidget/feedbackWidget.jsx

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+// ----- Imports ----- //
+
+// $FlowIgnore - required for hooks
+import React, { useState } from 'react';
+import { ThemeProvider } from 'emotion-theming';
+
+import { Button, LinkButton, buttonBrand } from '@guardian/src-button';
+import { SvgCross } from '@guardian/src-icons';
+import { sendTrackingEventsOnClick } from 'helpers/subscriptions';
+import { clickedCss, wrapper, buttonStyles, feedbackLink, header } from './feedbackWidgetStyles';
+import { SvgThumbsUp } from './thumbsUp';
+import { SvgThumbsDown } from './thumbsDown';
+
+function FeedbackWidget() {
+
+  const [clicked, setClicked] = useState({ positive: false, negative: false });
+  const positiveButtonCss = clicked.positive ? clickedCss : null;
+  const negativeButtonCss = clicked.negative ? clickedCss : null;
+
+  return (
+    <aside css={wrapper}>
+      <header css={header}>
+        <h4>{clicked.negative ? 'What can we improve?' : 'Is this page helpful?'}</h4>
+        {clicked.negative || (
+          <span>
+            <ThemeProvider theme={buttonBrand}>
+              <Button
+                priority="subdued"
+                size="small"
+                hideLabel
+                icon={<SvgThumbsUp />}
+                cssOverrides={[positiveButtonCss, buttonStyles]}
+                onClick={() => {
+              sendTrackingEventsOnClick({
+                id: 'landing_feedback_positive',
```

Comment:
I'd make this id as descriptive as possible for when you've forgotten what is is and are trying to write a LIKE clause to find it maybe something like 'ds_landing_page_survey_positive'

Reply:
done 👍